### DISCUSSION TITLE
Running new-backend on Windows machines with Node > 16.x

### DISCUSSION THREAD

**Discussion by perosoderstrom** [hajkmap/Hajk#906] [Q&A] [answered]
**Describe the bug**
Error when running `npm run dev` in new-backend:
**_'babel-node' is not recognized as an internal or external command,_**

**To Reproduce**

git clone . . .
git checkout develop
cd new-backend
npm install
npm run dev => Error

**Expected behavior**
A running nodejs backend

**Desktop (please complete the following information):**

- OS: Win10
- node -v = v16.13.0 (Latest LTS
- npm -v => 8.1.0

**Additional context**
Can this be related to latest changes in https://github.com/hajkmap/Hajk/commit/beba93ae3b506199d4a9b631c477e858d6f0ebc4 ?

**Comment by Hallbergs** [comment, marked as answer]
I've seen this issue on Windows-machines as well. Seems to be Node > 16.x related. I fixed it by changing the following:
`"dev": "nodemon server --exec babel-node --config .nodemonrc.json",` to
`"dev": "nodemon server --exec ./node_modules/.bin/babel-node --config .nodemonrc.json",`

A more long-term solution is coming up!

**Comment by Hallbergs** [comment]
It seems like Nodemon cannot find the packages for some reason. Use the fix above until we've figured out how to avoid this.

**Comment by Hallbergs** [comment]
For clarity:
![image](https://user-images.githubusercontent.com/43059093/141444220-d59c9d77-4d03-4801-a7cc-63f6c6ba2b40.png)
The change is required in `package.json` on the line above.

**Comment by perosoderstrom** [comment]
Thanks @Hallbergs ! Now it works!

**Comment by Hallbergs** [comment]
@perosoderstrom I pushed the fix to develop so that you don't have to remember to add the line yourself :) Hopefully we can add a more long-term solution soon!

**Reply by perosoderstrom** [reply]
Thanks @Hallbergs ! Very good!
